src/nidcpower/metadata/functions_addon.py

```diff
@@ -154,4 +150,96 @@
             'description': 'Returns the number of channels.',
         },
     },
+    # Public function that wraps driver function but returns datetime object instead of individual items
+    'GetLastExtCalLastDateAndTime': {
+        'codegen_method': 'public',
+        'returns': 'ViStatus',
+        'python_name': 'get_ext_cal_last_date_and_time',
+        'real_datetime_call': 'GetExtCalLastDateAndTime',
+        'method_templates': [
+            { 'session_filename': 'datetime_wrappers', 'documentation_filename': 'default_method', 'method_python_name_suffix': '', },
+        ],
+        'parameters': [
+            {
+                'direction': 'in',
+                'enum': None,
```

Comment:
Does our generator still force us to specify enum None? Should be the default...

Reply:
No, removed.